### PR TITLE
Cleanup "Working with containers" page.

### PR DESCRIPTION
**Pull request by charleswhchan**
- Cleanup introduction
- Update output of `docker version`
- Cleanup explaination of `docker inspect`
- Change 'deleting' to 'removing' for consistency

Signed-off-by: Charles Chan charleswhchan@users.noreply.github.com

### CODE REVIEW

File: docs/userguide/usingdocker.md [hunk 1]
```diff
@@ -43,18 +43,18 @@ version information on the currently installed Docker client and daemon.
...
-    Git commit (client): cc3a8c8
-    Server version: 0.8.0
-
-    Git commit (server): cc3a8c8
-    Go version (server): go1.2
-
-    Last stable version: 0.8.0
-
+  
+    Client version: 1.7.1
```
Comment (moxiegirl) [inline docs/userguide/usingdocker.md]: @charleswhchan Thank you for the contribution.  I noticed the format of this command has changed.

```
Client:
 Version:      1.8.0-dev
 API version:  1.21
 Go version:   go1.4.2
 Git commit:   0f49141
 Built:        Tue Aug  4 17:42:34 UTC 2015
 OS/Arch:      linux/amd64

Server:
 Version:      1.8.0-dev
 API version:  1.21
 Go version:   go1.4.2
 Git commit:   0f49141
 Built:        Tue Aug  4 17:42:34 UTC 2015
 OS/Arch:      linux/amd64
```

Comment (charleswhchan) [inline docs/userguide/usingdocker.md]: Yes, indeed. :smile:

Comment (charleswhchan) [inline docs/userguide/usingdocker.md]: Changes commit: Updated to latest format.

**Comment by thaJeztah** [comment]
LGTM, thanks!

**Comment by moxiegirl** [comment]
LGTM

**Pull request merged**